packages/circuit-ui/components/ListItemGroup/ListItemGroup.spec.tsx

```diff
@@ -102,6 +103,62 @@ describe('ListItemGroup', () => {
       });
       expect(wrapper).toMatchSnapshot();
     });
+
+    it('should render a ListItemGroup with interactive items', () => {
+      const { container } = renderListItemGroup(render, {
+        ...baseProps,
+        items: baseProps.items.map((item) => ({
+          ...item,
+          onClick: jest.fn(),
+        })),
+      });
+
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should render the focused item in a ListItemGroup with interactive items', () => {
+      const { container } = renderListItemGroup(render, {
+        ...baseProps,
+        items: baseProps.items.map((item) => ({
+          ...item,
+          onClick: jest.fn(),
+        })),
+      });
+
+      act(() => {
+        userEvent.tab();
+        userEvent.tab(); // blur first and focus second item
+      });
+
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should render the selected item in a ListItemGroup with interactive items', () => {
+      const { container } = renderListItemGroup(render, {
+        ...baseProps,
+        items: baseProps.items.map((item) => ({
+          ...item,
+          onClick: jest.fn(),
+          selected: item.key === 1,
+        })),
+      });
+
+      expect(container).toMatchSnapshot();
```

Comment:
I updated some of the tests to target just the selected list item but I'm still using `toMatchSnapshot()`. `toHaveStyle()` is good for some cases but is not always suitable for more complex scenarios. 
I also added a `toHaveAttribute("aria-pressed", "true")` test in the `accessibility` group.
Hopefully that slimmed down the specs a bit. I'll try to avoid component-wide snapshots in the future where possible.